src/com/intellij/idea/plugin/hybris/system/java/codeInsight/hints/LoggerInlayHintsProvider.kt

```diff
@@ -85,25 +86,40 @@ class LoggerInlayHintsProvider : JavaCodeVisionProviderBase() {
 
                 val range = InlayHintsUtils.getTextRangeWithoutLeadingCommentsAndWhitespaces(targetElement)
 
-                val text = RichText("[y] log level")
-                CxLoggerAccess.getInstance(project)
+                val logger = CxLoggerAccess.getInstance(project)
```

Comment:
Simplify retrieval of the logger names per `PsiClass` & `PsiPackageStatement` by using:
```kotlin
PsiTreeUtil.findChildrenOfAnyType(psiFile, PsiClass::class.java, PsiPackageStatement::class.java)
    .mapNotNull {
        val fqn = when (it) {
            is PsiClass -> FqnUtil.elementToFqn(it, editor)
            is PsiPackageStatement -> it.packageName
            else -> null
        } ?: return@mapNotNull null
        it to fqn
    }
```